setup.py

```diff
@@ -25,7 +25,9 @@
 version = "1.0.0"
 release_status = "Development Status :: 5 - Production/Stable"
 dependencies = [
-    "google-api-core[grpc] >= 1.14.0, < 2.0.0dev",
+    "google-api-core[grpc] >= 1.22.0, < 2.0.0dev",
+    "proto-plus >= 0.4.0",
+    "libcst >= 0.2.5",
     'enum34; python_version < "3.4"',
```

Comment:
'enum34; python_version < 3.4' is no longer needed since python is not required to >= 3.6

Reply:
sure, removed.